### PR TITLE
Persistence config

### LINKED ISSUE #32: [BUG][Opensearch] duplicate persistence config / storageClass option
**Describe the bug**
The configuration of the statefulset's persistence is done in two 
places, in `.Values.persistence` and in `.Values.volumeClaimTemplate` and should be
done in a single place instead, as is done in most other Helm Charts.
This also allows a more obvious place to enter a value for the `storageClass` to be used for the PVC.

**To Reproduce**
1. Go to the values.yaml file and check both config options

**Expected behavior**
only one place for PVC config should exist.

**Component**
peristence

**Host/Environment (please complete the following information):**
 - Helm v3.5.3
 - Version 1.0.0 of the tChart

### PR DESCRIPTION
**Pull request by belfhi**
### Description
merge volumeClaimTemplate options into 
values.persistence configuraion.
 
### Issues Resolved
fixes #32 
 
### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: Helm/opensearch/values.yaml [hunk 1]
```diff
@@ -163,7 +157,18 @@ persistence:
   enabled: true
   labels:
     # Add default labels for the volumeClaimTemplate of the StatefulSet
-    enabled: false
+    enabled: false
```
Comment (DandyDeveloper) [inline Helm/opensearch/values.yaml:160]: May be worth removing this? I'm not sure why this exists in the first place.

**Review by DandyDeveloper — APPROVED** [review]
One unrelated comment, LGTM!

File: Helm/opensearch/values.yaml [hunk 2]
```diff
@@ -163,7 +157,18 @@ persistence:
   enabled: true
   labels:
     # Add default labels for the volumeClaimTemplate of the StatefulSet
-    enabled: false
+    enabled: false      
+  ## Open Distro master Persistent Volume Storage Class
```
Comment (TheAlgo) [inline Helm/opensearch/values.yaml:161]: Nit: Lets change it to OpenSearch now

**Review by TheAlgo — CHANGES_REQUESTED** [review]

**Review by TheAlgo — APPROVED** [review]

**Comment by TheAlgo** [comment]
LGTM!

File: Helm/opensearch/values.yaml (hunk 1 above)
Comment (peterzhuamazon) [reply Helm/opensearch/values.yaml]: Is this issue resolved as of now? @DandyDeveloper @belfhi?

**Review by bbarani — APPROVED** [review]

File: Helm/opensearch/values.yaml (hunk 1 above)
Comment (peterzhuamazon) [reply Helm/opensearch/values.yaml]: Seems like there is no change to this anymore.

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**